near/test/sdk.ts

```diff
@@ -0,0 +1,874 @@
+      JSON.stringify(await masterAccount.getAccountBalance())
+  );
+
+  let userKey = nearUtils.KeyPair.fromRandom("ed25519");
+  keyStore.setKey(
+    config.networkId as string,
+    config.userAccount as string,
+    userKey
+  );
+  let user2Key = nearUtils.KeyPair.fromRandom("ed25519");
+  keyStore.setKey(
+    config.networkId as string,
+    config.user2Account as string,
+    user2Key
+  );
+
+  console.log(
+    "creating a user account: " +
+      config.userAccount +
+      " with key " +
+      userKey.getPublicKey()
+  );
+
+  await masterAccount.createAccount(
+    config.userAccount as string,
+    userKey.getPublicKey(),
+    new BN(10).pow(new BN(27))
+  );
+  const userAccount = new nearAccount(
+    near.connection,
+    config.userAccount as string
+  );
+
+  console.log(
+    "creating a second user account: " +
+      config.user2Account +
+      " with key " +
+      user2Key.getPublicKey()
+  );
+
+  await masterAccount.createAccount(
+    config.user2Account as string,
+    user2Key.getPublicKey(),
+    new BN(10).pow(new BN(27))
+  );
+  const user2Account = new nearAccount(
+    near.connection,
+    config.user2Account as string
+  );
+
+  console.log(
+    "Creating new random non-wormhole token and air dropping some tokens to myself"
+  );
+
+  let randoToken = nearProviders.getTransactionLastResult(
+    await userAccount.functionCall({
+      contractId: "test.test.near",
+      methodName: "deploy_ft",
+      args: {
+        account: userAccount.accountId,
+      },
+      gas: new BN("300000000000000"),
+    })
+  );
+
+  let token_bridge = CONTRACTS.DEVNET.near.token_bridge;
+  let core_bridge = CONTRACTS.DEVNET.near.core;
+
+  console.log("Setting up algorand wallet");
+
+  let algoCore = BigInt(CONTRACTS.DEVNET.algorand.core);
+  let algoToken = BigInt(CONTRACTS.DEVNET.algorand.token_bridge);
+
+  const tbAddr: string = getApplicationAddress(algoToken);
+  const decTbAddr: Uint8Array = decodeAddress(tbAddr).publicKey;
+  const aa: string = uint8ArrayToHex(decTbAddr);
+
+  const algoClient: algosdk.Algodv2 = getAlgoClient();
+  const tempAccts: Account[] = await getTempAccounts();
+  const numAccts: number = tempAccts.length;
+
+  const algoWallet: Account = tempAccts[0];
+
+  console.log("Creating USDC on Near");
+
+  let ts = new TestLib();
+  let seq = Math.floor(new Date().getTime() / 1000);
+  let usdcvaa = ts.hexStringToUint8Array(
+    ts.genAssetMeta(
+      ts.singleGuardianPrivKey,
+      0,
+      1,
+      seq,
+      "4523c3F29447d1f32AEa95BEBD00383c4640F1b4".toLowerCase(),
+      1,
+      8,
+      "USDC",
+      "CircleCoin"
+    )
+  );
+
+  seq = seq + 1;
+
+  let usdcp = _parseVAAAlgorand(usdcvaa);
+
+  //console.log(usdcp);
+
+  console.log("calling createWrappedOnNear to create usdc");
+
+  if (
+    (await getIsTransferCompletedNear(userAccount, token_bridge, usdcvaa)) ==
+    true
+  ) {
+    console.log("getIsTransferCompleted returned incorrect value (true)");
+    process.exit(1);
```

Comment:
FYI.  There is also the jest test()...expect()  framework, if you wanted to use that.
No changes requested by me here.

Reply:
Somebody needs to spend the time to airlift this entirely into the jest sdk tests...